avionics/statemachine.cpp

```diff
@@ -185,6 +185,7 @@ void stateMachine(float *altitude, float *delta_altitude, float *prev_delta_alti
                 if (delta_altitude_landed <= LAND_VELOCITY_THRESHOLD) { // Landed threshold based on velocity alone
                     land_count++;
                     if (land_count >= LAND_CHECKS) {
+                        stop_record;
```

Comment:
did this compile

Reply:
Also, are we going to remove the stop_record() in LANDED?

Reply:
should this be stop_record() instead?

Reply:
It actually compiles??!??  Throws a warning but that is alarming...
![image](https://user-images.githubusercontent.com/35241994/59653966-625ab100-9151-11e9-9665-560173378191.png)

Reply:
https://stackoverflow.com/a/31298367

Reply:
@jesseli2002 I'm fine with moving the stop_record() to the landing state transition. Not a big deal though because once it has stopped the video should be saved.